packages/dev/core/src/Materials/greasedLineSimpleMaterial.ts

```diff
@@ -12,7 +12,7 @@ import type { BaseTexture } from "./Textures/baseTexture";
 
 import "../Shaders/greasedLine.fragment";
 import "../Shaders/greasedLine.vertex";
-import "../Shaders/greasedLineNonCameraFacing.vertex";
+// import "../Shaders/greasedLineNonCameraFacing.vertex";

```

Comment:
```suggestion
```

Reply:
Hello, this is just a draft PR. Thanks anyway.

Reply:
Sorry, will wait for your "go" once it's ok for you before reviewing!